### ISSUE TITLE
Volatility for Glicko2Rating::unrated() should be specified

### ISSUE THREAD

**Issue by halvnykterist** [DenialAdams/glicko2#8]
Hi!

I'm using the library to calculate ratings for [http://ratingupdate.info/](http://ratingupdate.info/), and was running into an issue with deviations increasing quite rapidly over time, despite the system constant being very low.  After debugging a bit, I came to the conclusion that new glicko2 ratings had a volatility of 0.06 assigned was the problem.

In the original paper, the text is: 

> Set the player’s volatility to 0.06 (this value depends on the particular application).

This does mean that `From<GlickoRating>` on `Glicko2Rating` isn't really something that can be implemented, which has implications for the rest of the API.

**Comment by DenialAdams** [comment]
Thank you for the feedback, sorry that you had trouble!

I'll look into this today - I guess at a bare minimum we should create a method `Glicko2Rating::from_glicko_rating_with_volaitility`, and then we can decide if we want to also remove the From<GlickoRating> impl.

**Comment by halvnykterist** [comment]
Yeah, that sounds about right. I think a better "uninitialized" constructor would be good, too.
